R/utils.R

```diff
@@ -482,11 +482,9 @@ na2zero <- function(x) {
   x
 }
 
-sanitize_capture <- function(x) {
-  for(i in seq_along(x)) {
-    x[i] <- gsub("\\[|\\(", "_", x[i])
-    x[i] <- gsub("\\]|\\)", "",  x[i])
-  }
+sanitize_capture <- function(x, sep = "_") {
+  x <- gsub("\\[|\\(", sep, x, perl = TRUE)
+  x <- gsub("\\]|\\)", "",  x, perl = TRUE)
```

Comment:
The `perl = TRUE` switch wasn't needed, right?  (Just making sure I'm not missing something.)

Reply:
Right; wasn't needed.